packages/shared/src/graphql/keywords.ts

```diff
@@ -2,11 +2,17 @@ import { gql } from 'graphql-request';
 
 export type KeywordStatus = 'pending' | 'allow' | 'deny' | 'synonym';
 
+export type KeywordFlags = {
+  title?: string;
+  description?: string;
+};
+
 export interface Keyword {
   __typename?: string;
   value: string;
   occurrences: number;
   status: KeywordStatus;
+  flags?: KeywordFlags;
```

Comment:
is flags an array of flags or just object? if just one, maybe better name it flag 🙏

Reply:
We call them `flags` in all entities, its is set of properties.